Index: src/FTSize.cpp
===================================================================
--- src/FTSize.cpp	(revision 1266)
+++ src/FTSize.cpp	(working copy)
@@ -87,14 +87,7 @@
         return 0.0f;
     }
 
-    if(FT_IS_SCALABLE((*ftFace)))
-    {
-        return ((*ftFace)->bbox.yMax - (*ftFace)->bbox.yMin) * ((float)ftSize->metrics.y_ppem / (float)(*ftFace)->units_per_EM);
-    }
-    else
-    {
-        return static_cast<float>(ftSize->metrics.height) / 64.0f;
-    }
+    return static_cast<float>(ftSize->metrics.height) / 64.0f;
 }
 
 
